jackson/src/main/java/feign/jackson/JacksonEncoder.java

```diff
@@ -48,7 +49,7 @@ public JacksonEncoder(ObjectMapper mapper) {
   public void encode(Object object, Type bodyType, RequestTemplate template) {
     try {
       JavaType javaType = mapper.getTypeFactory().constructType(bodyType);
-      template.body(mapper.writerFor(javaType).writeValueAsString(object));
+      template.body(mapper.writerFor(javaType).writeValueAsBytes(object), StandardCharsets.UTF_8);
```

Comment:
The only change I think we should make here is use our `Util.UTF_8` constant for consistency.